config/database.docker.yml

```diff
@@ -0,0 +1,20 @@
+default: &default
+  adapter: postgresql
+  encoding: utf8
+  pool: 5
+  timeout: 5000
+  host: db
+  port: 5432
+  #socket: /var/run/mysqld/mysqld.sock
+
+development:
+  <<: *default
+  database: open_food_network_dev
+  username: ofn
+  password: f00d
+
+test:
+  <<: *default
+  database: open_food_network_test
+  username: ofn
+  password: f00d
```

Comment:
I'd rather work on https://github.com/openfoodfoundation/ofn-install/issues/387 (which I really want to do soon) which will provide the needed flexibility with ENV vars so we don't need to add purpose-specific database.yml like this one.

Reply:
Ok, let's wait for it then.

Reply:
#387 would be nice, but I don't think it needs to block this Docker setup..?

Reply:
We can go with https://github.com/openfoodfoundation/openfoodnetwork/pull/3954